ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -17,7 +17,7 @@ def Acosh(*, x, name="Acosh"):
 
 
 def Add(*, x, y, name="Add"):
-    return ivy.add(x, y)
+    return tf_frontend.add(x, y)
```

Comment:
if it's an alias, we should simply write `Add = tf_frontend.add` :)

Reply:
Oh my bad. I'll make the required changes